src/main/modules/nunjucks/index.ts

```diff
@@ -3,8 +3,9 @@ import * as express from 'express';
 import * as nunjucks from 'nunjucks';
 
 export class Nunjucks {
-  constructor(public developmentMode: boolean) {
+  constructor(public developmentMode: boolean, public i18next) {
```

Comment:
Are we just waiting for my PR for typing?

Reply:
yes